modules/chemical_reactions/doc/content/modules/chemical_reactions/activity_coefficients.md

```diff
@@ -0,0 +1,116 @@
+- Coefficients $A$, $B$ and $\dot{B}$ depend on temperature, and are given in the [chemical database](database.md).  At at 25$^{\circ}$C: $A=0.5092\,$mol$^{-1/2}$.kg$^{1/2}$, $B=0.3283\,$mol$^{-1/2}$.kg$^{1/2}$.$\mathring{A}^{-1}$ and $\dot{B}=0.0410\,$mol$^{-1}$.kg.
+
+### Neutral species
+
+When $z=0$, the activity coefficients may be computed using (Eqn(8.6) of [!cite](bethke_2007))
+\begin{equation}
+\log_{10}\gamma = aI + bI^{2} + cI^{3} + dI^{4} \ .
+\end{equation}
+Here $a$, $b$, $c$ and $d$ are functions of temperature, and are tabulated in [chemical databases](database.md) (assuming $I$ is measured in mol.kg$^{-1}$).  Also, the database must contain a keyword, such as $\mathring{a}=-0.5$, that instructs the solver to use this expression instead of the standard Debye-Huckel B-dot model (see Section 3.1.4 of [!cite](gwb_reference)).
+
+### Alternative formula
+
+Sometimes, the activity coefficients are computed using
+\begin{equation}
+\log_{10}\gamma = \dot{B}I \ .
+\end{equation}
+If so, the [chemical database](database.md) should contain a keyword $\mathring{a}=-1$ to instruct the solver to use this expression instead of the standard Debye-Huckel B-dot model  (see Section 3.1.4 of [!cite](gwb_reference)).
+
+## Activity of water
+
+The activity of water is (Eqns(8.7) and (8.8) of [!cite](bethke_2007))
+\begin{equation}
+\log a_{w} = -\frac{2I_{s}}{55.51} \left(
+1 - \frac{A\log 10}{\tilde{a}^{3}I_{s}}\left[ \hat{b}-2\log\hat{b} - \frac{1}{\hat{b}}\right] + \frac{\tilde{b}I_{s}}{2} + \frac{2\tilde{c}I_{s}^{2}}{3} + \frac{3\tilde{d}I_{s}^{3}}{4}
+\right) \ ,
+\end{equation}
+with
+\begin{equation}
+\hat{b} = 1 + \tilde{a}\sqrt{I_{s}} \ .
+\end{equation}
+In these expression
+
+- $\log$ is the natural logarithm
+- $I_{s}$ \[mol.kg$^{-1}$\] is the stoichiometric ionic strength
+- $55.51\,$mol.kg$^{-1}$ is the number of moles in a kg of water
+- $A$ \[kg$^{1/2}$.mol$^{-1/2}$\] is the standard Debye-Huckel $A$ parameter
+- $\tilde{a}$, $\tilde{b}$, $\tilde{d}$ and $\tilde{d}$ are temperature-dependent coefficients.
+
+The values of $\tilde{a}$, $\tilde{b}$, $\tilde{d}$ and $\tilde{d}$ are given in the [chemical database](database.md), assuming units of molality.  At 25$^{\circ}$C
+
+- $\tilde{a}=1.45397\,$kg$^{1/2}$.mol$^{-1/2}$,
+- $\tilde{b}=0.022357\,$kg.mol$^{-1}$,
+- $\tilde{c}=0.0093804\,$kg$^{2}$.mol$^{-2}$,
+- $\tilde{d}=-0.0005362\,$kg$^{3}$.mol$^{-3}$.
+
+## Bounds on ionic strength
+
+The expressions above are only valid to low ionic strength.  Therefore, $I$ and $I_{s}$ in the above formulae are bounded above by a user-defined parameter that defaults to $3\,$mol.kg$^{-1}$.
+
+## Davies model
+
+Here (Eqn(8.4) of [!cite](bethke_2007))
+\begin{equation}
+\log_{10}\gamma = -A z^{2} \left( \frac{\sqrt{I}}{1 + \sqrt{I}} - 0.3 I \right)
+\end{equation}
+This is valid for equilibrium solutions at 25$^{\circ}$C and up to ionic strengths of about 0.5 molal.  This will not be coded into the `geochemistry` module.
```

Comment:
Maybe 'isn't coded' rather than will not be (someone might do it one day?)

Reply:
Yep, ok